src/app/modules/emailbox/emailbox.component.html

```diff
@@ -13,7 +13,7 @@ <h6 class="inline">
             </div>
         </md-grid-tile>
     </md-grid-list>
-    <md-grid-list cols="1" rowHeight="24px">
+    <md-grid-list cols="1" rowHeight="24px" *ngIf="role !== 'Guest'" >
```

Comment:
make this a directive as well *ngIf="role !== 'Guest'"

Reply:
let it be ngIF only